CitationStyleLanguagePlugin.inc.php

```diff
@@ -335,24 +391,22 @@ public function getArticleTemplateData($hookName, $args)
      * @see Mendeley's mappings http://support.mendeley.com/customer/portal/articles/364144-csl-type-mapping
      *
      * @param Request $request
-     * @param Submission $article
+     * @param Submission $submission
      * @param string $citationStyle Name of the citation style to use.
      * @param Issue $issue Optional. Will fetch from db if not passed.
      * @param Publication $publication Optional. A particular version
      *
      * @return string
      */
-    public function getCitation($request, $article, $citationStyle = 'apa', $issue = null, $publication = null)
-    {
-        $publication ??= $article->getCurrentPublication();
-        $issue ??= Repo::issue()->get($publication->getData('issueId'));
```

Comment:
It looks like this line was removed. I think you'll need to restore it, but only call it in OJS.

Reply:
It has been added back.